jungle/ec2.py

```diff
@@ -116,17 +116,21 @@ def create_ssh_command(instance_id, instance_name, username, key_file, port, ssh
             target_instances = []
             for idx, i in enumerate(instances):
```

Comment:
Hey! So I went with `enumerate` for a couple reasons;

* I wanted to be consistent with the rest of the code, which also uses enumerate.
* From just trying this now, `ec2.Instance` is not iterable, so you _need_ to use `enumerate`

```
>>> import boto3
>>> ec2 = boto3.resource('ec2')
>>> instances = ec2.instances.filter(Filters=conditions)
>>> for idx,i in list(instances):
...      print str(idx) + ": " + i.public_ip_address
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'ec2.Instance' object is not iterable
>>> for idx,i in enumerate(instances):
...     print str(idx) + ": " + i.instance_id
...
0: i-022f5da367f528689
1: i-0e2cbdcf17c15acf9
```